docs/EN_US/ContainerizedHPCC/ContainerizedHPCCSystemsPlatform.xml

```diff
@@ -170,8 +170,8 @@
         enough that we recommend using a special tool for editing it.</para>
 
         <para>The <emphasis role="strong">values.yaml</emphasis> from the
-        default helm chart is relatively small and can be opened in any
-        editor, and/or modified via helm’s command-line overrides. It also is
+        default Helm chart is relatively small and can be opened in any
+        editor, and/or modified via Helm’s command-line overrides. It also is
```

Comment:
should probably be lower case if it refers to the command rather than the project